app-web/src/utils/helpers.js

```diff
@@ -183,5 +188,6 @@ export const sortDevhubTopicsAfterDesignSystem = topics => {
     return 0;
   });
 
-  return [designSystem].concat(sortedTopics);
+  //Add featured Cards, then Design System, then the rest of the topics
+  return [featuredCards].concat([designSystem]).concat(sortedTopics);
```

Comment:
just an fyi (dont have to change) `[featuredCards, designSystem].concat(sortedTopics);` would also work

Reply:
noted! thanks